testfm/helpers.py

```diff
@@ -8,14 +8,14 @@ def product():
         '-a "rpm -q satellite --queryformat=%{VERSION}" -o').read()
     project = sat_version.splitlines()[0].split(' ')[-1]
     if project.startswith('6.6'):
-        return 'sat66'
+        return ['sat66', '6.6']
```

Comment:
why this change?

Reply:
I need it to do proper teardown of tests. `'version': product()[1]  # Satellite minor version`

Reply:
Make sure it won't break run_only_on decorator.

Reply:
It doesn't.